akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/AskPattern.scala

```diff
@@ -146,14 +146,20 @@ object AskPattern {
     val ref: ActorRef[U] = _ref
     val future: Future[U] = _future
     val promiseRef: PromiseActorRef = _promiseRef
+
+    @InternalStableApi
+    private[akka] def ask[T](target: InternalRecipientRef[T], message: T, @unused timeout: Timeout): Future[U] = {
+      target ! message
+      future
+    }
   }
 
+  @InternalStableApi
```

Comment:
Does both of these need to be stable api, doesn't one just call the other? Did I ask this alread? :)

Reply:
Perhaps you did. Either I forgot to remove it or it got back when I was rebasing it. Originally I added it because this method is also instrumented to make it work for the previous versions of Akka. Now I think it's not needed because it won't be instrumented for future version and can be changed without breaking the internal spi. I'll remove it.